BoxOffice/Controller/MovieInformationViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  MovieInformationViewController.swift
+//  BoxOffice
+//
+//  Created by Dasan & Whales on 2023/08/14.
+//
+
+import UIKit
+import OSLog
+
+final class MovieInformationViewController: UIViewController {
+    private let boxOfficeManager = BoxOfficeManager()
+    private var movieInformation: MovieInformation?
+    private var movieCode: String?
+    private var posterImage: UIImage?
+    private let activityIndicatorView = UIActivityIndicatorView()
+    
+    init(movieCode: String) {
+        super.init(nibName: nil, bundle: nil)
+        self.movieCode = movieCode
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .systemBackground
+        
+        configureActivityIndicatorView()
+        startActivityIndicator()
+        loadData()
+    }
+    
+    private func loadData() {
+        guard let movieCode = movieCode else { return }
+        
+        boxOfficeManager.fetchMovieData(movieCode: movieCode){ result in
```

Comment:
p4;

띄어쓰기가 하나 누락된 거 같아요 👀
```suggestion
        boxOfficeManager.fetchMovieData(movieCode: movieCode) { result in
```

Reply:
수정하였습니다😂